integration/dockerfiles/Dockerfile_test_parent_dir_perms

```diff
@@ -0,0 +1,8 @@
+FROM busybox
+
+RUN adduser --disabled-password --gecos "" --uid 1000 user
+RUN mkdir -p /home/user/foo
+RUN chown -R user /home/user
+RUN chmod 700 /home/user/foo
+ADD Makefile /home/user/foo/Makefile
```

Comment:
Could you remove this line? We don't have a Makefile in the context which is causing the integration test build to fail.

Reply:
This line cannot be deleted because having ADD between chown cause this issue. Is there any file available in the environment?

Reply:
Ah, maybe we can use some URL to download a file because it’s ADD. let me update the PR.